ui/app/controllers/workspace/projects/project/app/deployments.ts

```diff
@@ -0,0 +1,27 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { Deployment } from 'waypoint-pb';
+import { action } from '@ember/object';
+export default class WorkspaceProjectsProjectAppDeployments extends Controller {
+  queryParams = ['destroyed']
+
+  @tracked destroyed = false;
+
+  get hasMoreDeployments() {
```

Comment:
In general we are following a pattern of really only using controllers for query params, see here: https://github.com/hashicorp/waypoint/tree/main/ui/app/controllers

You could potentially pull this out to the app-item/deployment component?

Reply:
(so the controller is still necessary but only tracks the query param)